utils/npm-install/npm-install.js

```diff
@@ -12,13 +12,20 @@ const getExecOpts = require("@lerna/get-npm-exec-opts");
 module.exports = npmInstall;
 module.exports.dependencies = npmInstallDependencies;
 
-function npmInstall(pkg, { registry, npmClient, npmClientArgs, npmGlobalStyle, mutex, stdio = "pipe" }) {
+function npmInstall(
+  pkg,
+  { registry, npmClient, npmClientArgs, npmGlobalStyle, mutex, stdio = "pipe", npmCiMode = false }
```

Comment:
This options block should include `subCommand = "install"` instead of `npmCiMode`